src/soc/ibm/power9/istep_10_10.c

```diff
@@ -574,24 +606,25 @@ static void phase1(const struct lane_config_row **pec_cfgs,
 	}
 }
 
-void istep_10_10(uint8_t *phb_active_mask, uint8_t *iovalid_enable)
+void istep_10_10(uint8_t chips, struct pci_info *pci_info)
 {
-	const struct lane_config_row *pec_cfgs[MAX_PEC_PER_PROC] = { NULL };
-
 	printk(BIOS_EMERG, "starting istep 10.10\n");
 	report_istep(10,10);
 
-	determine_lane_configs(phb_active_mask, pec_cfgs);
+	for (uint8_t chip = 0; chip < 8; chip++) {
```

Comment:
```suggestion
	for (uint8_t chip = 0; chip < 2; chip++) {
```

`pci_info` only has 2 elements so we shouldn't make the code able to read out of bounds, coverity scan could complain about it.

Reply:
Or even better, `#define` this value and use it where appropriate, in case there will ever be a board with more than 2 CPU sockets.